### PR TITLE
Fix dimensionality of log_std in GaussianMLPModule

### PR DESCRIPTION
**Pull request by krzentner**

### CODE REVIEW

**Review by ryanjulian — APPROVED** [review]

File: src/garage/torch/modules/gaussian_mlp_module.py [hunk 1]
```diff
@@ -121,12 +121,13 @@ def __init__(self,
         if self._std_parameterization not in ('exp', 'softplus'):
             raise NotImplementedError
 
-        init_std_param = torch.Tensor([init_std]).log()
+        self._init_std = torch.Tensor([init_std]).log()
+        log_std = torch.Tensor([init_std] * output_dim).log()
```
Comment (akashvelu) [inline src/garage/torch/modules/gaussian_mlp_module.py:125]: don't think a log() is necessary here, since it has already been taken in line 124?

**Review by akashvelu — APPROVED** [review]

File: src/garage/torch/modules/gaussian_mlp_module.py (hunk 1 above)
Comment (akashvelu) [inline src/garage/torch/modules/gaussian_mlp_module.py:125]: don't think a log() is necessary here, since it has already been taken in line 124?

**Review by akashvelu — CHANGES_REQUESTED** [review]

**Comment by avnishn** [comment]
@krzentner ptal at @akashvelu's comments

File: src/garage/torch/modules/gaussian_mlp_module.py (hunk 1 above)
Comment (akashvelu) [inline src/garage/torch/modules/gaussian_mlp_module.py:125]: don't think a log() is necessary here, since it has already been taken in line 124?

Comment (krzentner) [reply src/garage/torch/modules/gaussian_mlp_module.py:125]: Oh, good catch. Thanks!

Comment (krzentner) [reply src/garage/torch/modules/gaussian_mlp_module.py:125]: Oh, I see. Both of these lines used `init_std`, so it was actually fine. Thankfully we have tests that cover this line.

**Comment by krzentner** [comment]
The only untested on CI line in the diff is GPU-only, so I'm merging this.

**Pull request merged**